fairer-iOS/fairer-iOS/Network/DTO/RulesResponse.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RulesResponse.swift
+//  fairer-iOS
+//
+//  Created by 김규철 on 2023/03/05.
+//
+
+struct RulesResponse: Codable {
+    let ruleResponseDtos: [ruleData]?
+    let teamId: Int
+}
+
+struct ruleData: Codable {
```

Comment:
p4;
```suggestion
struct RuleData: Codable {
```
code convention을 맞추기 위해 struct는 대문자로 시작하는 게 좋을 것 같아요!

Reply:
https://github.com/fairer-iOS/fairer-iOS/pull/112/commits/f638deb5ffe8a664ed3e4a62c42ce1e3659e2a33